### DISCUSSION TITLE
Welche Daten liefert die Entität  Grid CT (R, S, T)

### DISCUSSION THREAD

**Discussion by Ingo-Vetter** [nathanmarlor/foxess_modbus#628] [Q&A]
Hallo zusammen, 

kann mir vielleicht jemand sagen, welche Werte die Entitäten Grid CT (R, S, T) zurück geben?

Viele Grüße
Ingo

**Comment by horstvogel** [comment]
Moin, habe das bei einem Fox ess noch nicht am Laufen, bin aber "auf der Zielgeraden" R S T, sind im Normalfall die 3 Phasen, also L1, L2,L3.
